src/core_atmosphere/physics/mpas_atmphys_driver_microphysics.F

```diff
@@ -204,6 +205,11 @@ subroutine allocate_microphysics(configs)
              if(.not.allocated(volg_p)) allocate(volg_p(ims:ime,kms:kme,jms:jme))
           endif
 
+          if (config_tempo_ml_nc_pbl) then
+             if(.not.allocated(cldfrac_p) ) allocate(cldfrac_p(ims:ime,kms:kme,jms:jme))
+             if(.not.allocated(qcbl_p) ) allocate(qcbl_p(ims:ime,kms:kme,jms:jme))
+          endif
+
```

Comment:
I stopped by (nextdoor) but you weren't around. I can't tell from this PR alone how the cldfrac_p and qcbl_p are getting populated after they become allocated. They must be, but I don't see it.

Reply:
I'm more accustomed to having these arrays populated in the driver, but I see there are many exceptions, so this is fine. No harm, no foul.